CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -124,7 +124,7 @@ public void reload() throws IOException{
                  		
 		this.isSaveCorefile = getProperty("main.testing.savecorefile", "FALSE").trim().toUpperCase().equals("TRUE");
 		this.bigSpaceDir = getProperty("main.testing.bigspace_dir", "").trim();
-		this.maxRetryCount = Integer.parseInt(getProperty("max.retry.count", "0").trim());
+		this.maxRetryCount = Integer.parseInt(getTestCaseMaxRetryCount());
```

Comment:
It's better to remove getTestCaseMaxRetryCount() and only keep getMaxRetryCount().
They are duplicated and confused to users.